src/event/sys/windows/parse.rs

```diff
@@ -61,19 +61,16 @@ pub(crate) fn handle_key_event(
 }
 
 fn handle_surrogate(surrogate_buffer: &mut Option<u16>, new_surrogate: u16) -> Option<char> {
-    match *surrogate_buffer {
-        Some(buffered_surrogate) => {
-            *surrogate_buffer = None;
-            std::char::decode_utf16([buffered_surrogate, new_surrogate])
-                .next()
-                .unwrap()
-                .ok()
-        }
-        None => {
-            *surrogate_buffer = Some(new_surrogate);
-            None
-        }
+    if new_surrogate < 0xDC00 {
```

Comment:
What does this value represent. A constant would help document this.

Reply:
Used `LOW_SURROGATE_FIRST` and also named the existing constants.